### PR TITLE
upgrading billing api to v5 and commenting breaking changes

### PR DESCRIPTION
**Pull request by rodriigovieira**

### CODE REVIEW

File: android/src/main/java/expo/modules/inapppurchases/BillingManager.java [hunk 1]
```diff
@@ -141,7 +141,7 @@ public void purchaseItemAsync(final String skuId, @Nullable final ReadableArgume
     ReadableArguments d = details != null ? details : new MapArguments();
     String oldPurchaseToken = d.getString("oldPurchaseToken");
     ReadableArguments accountIdentifiers = d.getArguments("accountIdentifiers");
-    Boolean isVrPurchaseFlow = d.getBoolean("isVrPurchaseFlow");
+    // Boolean isVrPurchaseFlow = d.getBoolean("isVrPurchaseFlow");
```
Comment (lukebrandonfarrell) [inline android/src/main/java/expo/modules/inapppurchases/BillingManager.java:144]: Is this no longer supported, is there another property which replaces it?

Comment (rodriigovieira) [reply android/src/main/java/expo/modules/inapppurchases/BillingManager.java:144]: I have no idea. Just got it from a comment. Do you know if this property is useful for something related to our project?

Comment (rodriigovieira) [reply android/src/main/java/expo/modules/inapppurchases/BillingManager.java:144]: https://github.com/expo/expo/issues/23920#issuecomment-1693813275

Comment (lukebrandonfarrell) [reply android/src/main/java/expo/modules/inapppurchases/BillingManager.java:144]: It's not I guess we can add it back if needed

**Review by lukebrandonfarrell — APPROVED** [review]

**Pull request merged**